infra/terraform/test-org/test-cleanup/cleanup.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+module "app-engine" {
```

Comment:
why app engine here??

Reply:
I've copied that block from `test/setup` section of the `scheduled-function` module. Without that block scheduler job creation fails with:
```
Error: Error creating Job: googleapi: Error 404: The project cft-project-manager-test-org does not contain an App Engine application. Please visit https://console.developers.google.com/appengine?project=cft-project-manager-test-org to to add an App Engine application.
```